Testnet release plan.md

```diff
@@ -0,0 +1,55 @@
+# Testnet versions
+
+This is the large task breakdown I propose we add to the next versions of testnet.
+There will be bug fixes and UX improvements as we discover them, which will be treated as high priority.
+
+Note that there is no protocol work listed here. These are the tasks that are orthogonal to the protocol.
+
+
+## V2
+1. Events subscription
+2. Connect to a well-known Ethereum testnet
+   a) support restarting of nodes
+3. Improvements to Management contract
+   a) Support withdrawals
+4. Mempool improvements
+   a) Remove failed transactions
+
+
+## V3
+1. Support a limited number of user operated nodes
+   a) Catching up robustness
+   b) Script robustness
+   c) Instructions
+   d) Try outside of Azure?
+2. Design and implement reveleation period
```

Comment:
Is this lower priority than some of the items in v4, v5?

Reply:
tbh, this is not strictly in priority order. It's stuff that needs to be done anyway. 

We might decide to go for a "light revelation" at this stage with a single period.